environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
All should be fine. New version inherits data from previous versions unless otherwise specified in `data-*.js`-files